modules/ROOT/pages/setup-payload-storage-API.adoc

```diff
@@ -1,29 +1,33 @@
 = Payload Storage API for Partner Manager
 
-Anypoint Partner Manager enables you to configure your own storage solution, such as database, Amazon S3, Azure Blob Storage, and so on to store transmission content, such as incoming and outgoing B2B message payloads.  
+Anypoint Partner Manager enables you to configure your own storage solution (for example, a database, Amazon S3, or Azure Blob Storage) to store transmission content, such as incoming and outgoing B2B message payloads.
 
-Before you create your first trading partner and message flow, you must configure your payload storage API within the host organization in Partner Manager.
+Before you create your first xref:configure-partner#create-partner[trading partner] and xref:configure-message-flow[message flow], you must configure your payload storage API within the host organization in Partner Manager.
 
-To store your content, you must implement an API that provides Anypoint Partner Manager with a standardized way to connect to your unique storage.
+To use your own storage solution for warehousing content, you must implement an API that provides Anypoint Partner Manager with a standardized way to connect to your storage.
 
-== Download and Deploy the Payload Storage API 
+== Download and Deploy the Payload Storage API
 
-Follow these steps to set up the payload storage API:
+=== Prerequisites
 
-. Download these APIs from Anypoint Exchange:
-+
-https://www.mulesoft.com/exchange/com.mulesoft.b2b/partner-manager-content-storage-api[Partner Manager Content Storage API]
+Before you can deploy the Payload Storage API, you must first access and download the following APIs from Anypoint Exchange:
+
+. Access and download the following APIs from Anypoint Exchange:
```

Comment:
We don't need an intro and a step here, and there is only one step, anyway.
This is moot; this is not a prerequisite to downloading and deploying, because it *is* downloading.
Looks to me like this topic should be...:

==Download and Deploy...
intro
1. Download the following APIs .... (note plural; should the title etc. be plural "APIs"?)
2. Implement the API (APIs?)
3. Deploy the API, making note of the following values:
.
.
.
== Add the API (how is this different from deploying?)

Reply:
Done!

Updated - ==Add the Payload Storage API to the Host Profile